pkg/cmd/package/upload/upload.go

```diff
@@ -0,0 +1,182 @@
+package upload
+
+import (
+	"errors"
+	"fmt"
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/constants/annotations"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/util"
+	"github.com/OctopusDeploy/cli/pkg/util/flag"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/newclient"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/packages"
+	"github.com/spf13/cobra"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// TODO Outstanding questions for package upload:
+// - What should the response be for outputformat basic and json?
+// - Basic output: I have "Successfully uploaded package x-1.0.0.zip" when the package was created/overwritten, and "Successfully processed package x-1.0.0.zip" when it was ignored. Is this right?
```

Comment:
`octo` will inform you if the package is NOT overwritten. We should do the same.

Reply:
ok cool I'll have a play with `octo` and put in some similar wording